### PR TITLE
fix #2050: Improve slack icon contrast ratio

### LINKED ISSUE #2050: Improve color contrast for slack icon in dark mode
**What happened**:
image
In about footer, the slack icon is hard to see in dark mode.
**What should have happened**:

**How to reproduce it (as precise as possible)**:

**Anything else we need to know?**:

**Environment**:

- OS:
- Browser:

### PR DESCRIPTION
**Pull request by yuanLeeMidori**
## Issue This PR Addresses
Fixes: #2050 

## Type of Change

- [x] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description

I wasn't able to fill color to the current slack svg, so I use the slack icon from react-icon instead. 

Desktop
chrome-capture (6)

Mobile
chrome-capture (7)

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Review by PedroFonsecaDEV — DISMISSED** [review]

File: src/web/src/components/AboutFooter.tsx [hunk 1]
```diff
@@ -3,8 +3,7 @@ import { Box, Divider, Grid, Typography, useMediaQuery } from '@material-ui/core
 import GitHubIcon from '@material-ui/icons/GitHub';
 import GitHubContributorCard from './GitHubContributorCard';
 import Logo from './Logo';
-
-const slackLogo = '/slack-logo.svg';
```
Comment (humphd) [inline src/web/src/components/AboutFooter.tsx:7]: Can we delete this file if you're not using it here (i.e., is it used anywhere else?  If not, remove it).

File: src/web/package.json [hunk 2]
```diff
@@ -25,6 +25,7 @@
     "next-pwa": "^5.1.3",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-icons": "^4.2.0",
```
Comment (humphd) [inline src/web/package.json:28]: Fascinating, I thought we had this in already.  It's a great library.

**Review by humphd — CHANGES_REQUESTED** [review]

**Comment by humphd** [comment]
cc @birtony, it looks like this is failing snapshot tests.

File: src/web/src/components/AboutFooter.tsx (hunk 1 above)
Comment (yuanLeeMidori) [reply src/web/src/components/AboutFooter.tsx:7]: I'll look into it and delete it if we don't need it.

**Review by PedroFonsecaDEV — APPROVED** [review]

**Review by chrispinkney — DISMISSED** [review]
Nice!

**Review by PedroFonsecaDEV — APPROVED** [review]

**Pull request merged**